developers/weaviate/configuration/persistence.md

```diff
@@ -104,7 +104,7 @@ changed the thresholds) you can use the [Shards API](../api/rest/schema.md#inspe
 :::info Available from version `v1.21`
 :::
 
-You can choose between `mmap` (DEFAULT) and `pread` functions to access virtual memory by setting the `PERSISTENCE_LSM_ACCESS` environment variable.
```

Comment:
@daveatweaviate and @sebawita Here is an example of a diff over a long line (148 chars originally). Seems fine to me w/ a soft wrap.